src/main/kotlin/com/openlattice/chronicle/postgres/ResultSetAdapters.kt

```diff
@@ -341,5 +343,14 @@ class ResultSetAdapters {
         fun storage(rs: ResultSet): String {
             return rs.getString(STORAGE.name)
         }
+
+        @Throws(SQLException::class)
+        fun participant(rs: ResultSet): Participant {
+            return Participant(
+                rs.getString(PARTICIPANT_ID.name),
+                Candidate(id = rs.getObject(CANDIDATE_ID.name, UUID::class.java)),
```

Comment:
May want to make candidate fields configurable.

Reply:
Doing so would require checking permissions for the caller because `Candidate` is a securable object.

Reply:
Good point, it might be best to let frontend call candidates api for details in that case.